### DISCUSSION TITLE
Request: Turkish Translation

### DISCUSSION THREAD

**Discussion by dora-meowmeow** [edrlab/thorium-reader#2645] [Ideas]
Hello, I've read the wikipage regarding translations and, it seems that I need to contact someone to add a new language? I would like to translate Thorium into Turkish. Please let me know steps I should take to start this process.

**Comment by horus68** [comment]
@danielweck can you create this new language and assigne this user to it?
https://gitlocalize.com/repo/9626

**Reply by danielweck** [reply]
@gautierchomel are you able to help with GitLocalize at the moment? Otherwise let us know and we will try to set things up for Turkish :) Thank you!

**Comment by gautierchomel** [comment]
@catdotjs, thank you for your involvement and my apologies for the delay, I missed the issue.

Currently gitlocalize server replies with a code 500 and does not succeed in creating the new language. I'll proceed as soon as it is fixed.

**Comment by gautierchomel** [comment]
@catdotjs, I was able to set up gitlocalize to add Turkish. 

You are now moderator for Turkish (tr) language and once logged to gitlocalize with your GitHub account, should be able to start localizing on the dedicated page https://gitlocalize.com/repo/9626/tr/src/resources/locales/en.json

Short instructions are available on the [Thorium Localization wiki page](https://github.com/edrlab/thorium-reader/wiki/Localization-(l10n)-language-translations)

I'll let you close this discussion once you are considering the set up functional.

**Reply by dora-meowmeow** [reply]
Alright thank you

**Comment by dora-meowmeow** [comment]
@gautierchomel Where can I ask for further context regarding some text I'm translating? I've left a comment on them in gitlocalize not sure if that is enough?

**Comment by gautierchomel** [comment]
@catdotjs I tried to answer in gitlocalize. I think it is the right place for that, but I get no notification when you add a comment, so keep on notifying me here.

**Comment by dora-meowmeow** [comment]
@gautierchomel have several more questions. Some are critical.

**Comment by gautierchomel** [comment]
@catdotjs I just answered. 

That's a very interesting exercise; I have added to my todo list to build a context table to help further work.

**Comment by dora-meowmeow** [comment]
@gautierchomel heya

is it possible for me to be no longer considered translator? I haven't been using thorium for awhile and not really interested in translating it any further.

**Comment by gautierchomel** [comment]
Hey @catdotjs, yes, of course. If you agree, I'll keep a reference to "catdotjs" in the localisation documentation without the @ before, so you are still mentioned for credit but will no longer get notifications for future calls to translators. 

Many thanks for your great work and participation; I remember that we built the localisation context table based on your feedback.

**Reply by dora-meowmeow** [reply]
Sorry for incredibly late response but I am 100% okay with that. I'm very glad that I've helped in any way. I wish you all the best <3
